docs/orchestration/README.md

```diff
@@ -3,20 +3,20 @@
 Minio is a cloud-native application designed to scale in a sustainable manner in multi-tenant environments. Orchestration platforms provide perfect launchpad for Minio to scale.  
 
 ## What is cloud-native architecture?
-Cloud-native applications offer easy scale-out and hardware decoupling. They often have small footprints with no baggage of monolithic application era. This means, while the application itself appears to be pretty lightweight, it scales-out sustainably to achieve high availability and resilience. 
+Cloud-native applications offer easy scale-out and hardware decoupling. They often have small footprints with no baggage of monolithic application era. This means, while the application itself appears to be pretty lightweight, it scales-out sustainably to achieve high availability and resilience.
 
 Take for example a web server. If running in a cloud environment, you wouldn't think twice before launching multiple instances of the server to handle extra load. This is because modern web servers don't have any hardware or software constraints. They scale-out to handle the extra load.
 
-While storage has been traditionally thought of as complicated, Minio changes that with its cloud-native architecture. As a true scale-out system, Minio scales with your tenants. As and when you need more storage, you can add a new Minio instance bound to a tenant. 
+While storage has been traditionally thought of as complicated, Minio changes that with its cloud-native architecture. As a true scale-out system, Minio scales with your tenants. As and when you need more storage, you can add a new Minio instance bound to a tenant.
 
 > In a cloud native environment, scalability is not a function of the application but the orchestration platform.
 
 ## Orchestration platforms
 
-Below is the list of documented orchestration platforms for Minio server: 
+Below is the list of documented orchestration platforms for Minio server:
 
 | Orchestration platforms|
 |:---|
-| [`Kubernetes`](./kubernetes) |
-| [`Docker Swarm`](./docker-swarm) |
-| [`DC/OS`](./dcos) |
+| [`Kubernetes`](./kubernetes/README.md) |
```

Comment:
You need to point to full URL not just this like this 

[`Kubernetes`](https://raw.githubusercontent.com/minio/minio/master/docs/orchestration/kubernetes/README.md)